queue/abstract.lua

```diff
@@ -481,6 +481,18 @@ function method.create_tube(tube_name, tube_type, opts)
     return self
```

Comment:
commit message: seems it should be `The patch adds` instead of `The patch added`

Reply:
Updated.